src/Api/Controllers/GroupsController.cs

```diff
@@ -58,10 +58,10 @@ public async Task<GroupDetailsResponseModel> GetDetails(string orgId, string id)
         public async Task<ListResponseModel<GroupResponseModel>> Get(string orgId)
         {
             var orgIdGuid = new Guid(orgId);
-            var canAccess = _currentContext.ManageGroups(orgIdGuid) ||
-                _currentContext.ManageAssignedCollections(orgIdGuid) ||
-                _currentContext.ManageAllCollections(orgIdGuid) ||
-                _currentContext.ManageUsers(orgIdGuid);
+            var canAccess = await _currentContext.ManageGroups(orgIdGuid) ||
+                            await _currentContext.ManageAssignedCollections(orgIdGuid) ||
+                            await _currentContext.ManageAllCollections(orgIdGuid) ||
+                            await _currentContext.ManageUsers(orgIdGuid);
```

Comment:
I think the way this was indented back before is still the appropriate formatting